Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM docker.io/library/golang:1.19.4-alpine3.17 as builder
```

Comment:
suggestion: build with the go version set on go.mod (similar to https://github.com/k8snetworkplumbingwg/kubemacpool/pull/384)

Reply:
The `go <version>` in the `go.mod` file is the minimum go version required to build the project.
Because go is backwards compatible, this number could stay low (if you don't use newer language features), while the tooling versions could increase.

Reply:
like I said - not a must. I just like being up to date I guess :)

Reply:
I intend to bump the official go image versions.
The `go <version>` in `go.mod` does not always have to change with it.

Reply:
why not?

Reply:
Lets say you are using go 1.17, and don't use a feature from go 1.18.
The tooling the project uses could be bumped to the latest versions, but the minimal version that can be used to build the code, doesn't necessarily have to change.
Although in practice the minimum go version will probably be bumped because we use third party libraries.

Reply:
This is not fully correct.
The minimum version needs to be bumped because there are always SRE/s.
And the support of old Go versions is discontinued (i.e. no new security fixes will be released).

Having this number is also a declaration from your part.
As this is not a library, it has no implications on others depending on it, but it does have implications if someone looks to see how secure it is kept.